server/db-middleware/submission-api.js

```diff
@@ -819,21 +848,23 @@ apiModule.sendCheckingResults = function (checkingId, res) {
         .findById(check.submissionId)
         .select('answers _id');
     })
-    .then(submission => Promise.all(submission.answers.map((el) => {
-      if (el.checking && el.questionId == question) {
-        const elReassign = {};
-        elReassign.checking = 'false';
-        elReassign.questionId = el.questionId;
-        elReassign.answ = el.answ;
-        elReassign.result = res;
-        return elReassign;
-      }
-      return el;
-    })))
+    .then((submission) => {
+      updateQuestion(question, res);
+      return Promise.all(submission.answers.map((el) => {
+        if (el.questionId == question) {
+          const elReassign = {};
```

Comment:
why not const elReassign = { checking: ..., .........}

Reply:
the you can write your whole promise like el.questionId == question ? { checking: ......, } : el;